db/versions/0055.yml

```diff
@@ -0,0 +1,27 @@
+version: 55
+description: add upload db fn to objects service
+methods:
+  upload_object:
```

Comment:
This doesn't actually *upload* anything.  Maybe `create_object` is a better name?